src/stability_sdk/client.py

```diff
@@ -296,18 +298,25 @@ def generate(
         :param height: Height of the generated image
         :param steps: Number of steps to run the diffusion process
         :param seed: Random seed for the starting noise
+        :param samples: Number of samples to generate
         :param cfg_scale: Classifier free guidance scale
         :param sampler: Sampler to use for the diffusion process
         :param init_image: Initial image to use
         :param init_strength: Strength of the initial image
         :param init_noise_scale: Scale of the initial noise
         :param mask: Mask to use (0 for pixels to change, 255 for pixels to keep)
         :param masked_area_init: How to initialize the masked area
+        :param mask_fixup: Whether to restore the unmasked area after diffusion
         :param guidance_preset: Preset to use for CLIP guidance
         :param guidance_cuts: Number of cuts to use with CLIP guidance
         :param guidance_strength: Strength of CLIP guidance
-        :return: The generated image
+        :return: dict mapping artifact type to data
```

Comment:
Shouldn't we preserve current `generate` behaviour for existing sdk users with additional `return_all=False` param like you did in your previous PR?

Reply:
The Api class is new to animation branch so there are no existing users of it yet besides animation notebook. The complicating factor for returning single image is it now supports # samples (with intention of deprecating StabilityInference and mapping its calls to this new Api)